html/excercise_two.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Excersie Two</title>
+  </head>
+  <body>
+    <h2>New Task</h2>
+    <p>Create a new monthly task and keep track of it.</p>
+    <a href="">Back</a>
+    <br>
+    <br>
+    <form>
+      <label>Name: <br></label>
+      <input><br>
+      <label>Description: <br></label>
+      <input><br><br>
+    </form>
+    <button>create</button>
```

Comment:
as this is a form, the create button should be also an input with `type="submit"`

```suggestion
    <input type="submit" value="Create" />
```

something like this, the way you did it you are not submitting the form, also it must be inside the form tag